Inventory.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/*
+*  Inventory.java sets the carSeatCovers list and adds a car into the list. Creates a list for themes.
+* method for comparing the user entry to the products in the inventory.txt file */
+
+public class Inventory {
```

Comment:
Just a note on some OOP concepts. Not all classes or objects are as physical or literal as a product. This `Inventory` class is a really good example of a class that doesn't describe a 'thing', but more of a 'person'.

I find it really useful to imagine these kinds of classes as lazy employees in a workplace. Instead of `Inventory` being a physical object, think of it more as a person that you can ask questions and that will do tasks for you. In this case, you can ask them to add a product to the inventory, which they will happily do because it's their job. 

But if you ask them, "hey, can you format this CarSeatCover into a description for me?" they won't know how to do that, because it's not their job to do. `Inventory` will find and hand you items from the warehouse, but they don't know what to do with them. 

I'm sure you've been taught this about OOP already, but I thought I'd share this 'employee' analogy that helps me think through some OOP stuff when things get more abstract. This kind of thinking is hard, and if you choose to get into OOP in your career, it will take time to learn how to think this way (which I am still in the process of doing).